docs/future-work/learning-module-improvements/use-off-object-observations.md

```diff
@@ -1,12 +1,51 @@
 ---
 title: Use Off-Object Observations
-rfc: https://github.com/thousandbrainsproject/tbp.monty/pull/425/files
+description: Ensure that off-object observations are processed by LMs, resulting in evidence updates.
+rfc: optional
+estimated-scope: medium
+improved-metric: speed, numsteps, accuracy
+output-type: experiments, analysis, PR
+skills: python, research, monty
```

Comment:
It sounds like most of the experimenting and analysis is in the "Use Out of Model Movements" part. This element here is mostly about updating code and making sure everything still works as expected. I've used `experiments` for cases where new experiment configs would be added. Do you think this would be the case here? Maybe we also add `refactoring` to the skills?

Reply:
Ah ok I think I misread your comment the other way around. Let me read it again and think if there is anything else to change.

Reply:
Thanks @vkakerbeck , wording has been updated to generally make the sequence and outcomes clearer. Let me know what you think.